centreon/src/Core/HostGroup/Infrastructure/Serializer/HostGroupResponseNormalizer.php

```diff
@@ -23,14 +23,18 @@
 
 namespace Core\HostGroup\Infrastructure\Serializer;
 
-use Core\HostGroup\Application\UseCase\FindHostGroups\Response\HostGroupResponse;
+use Core\HostGroup\Application\UseCase\FindHostGroups\HostGroupResponse;
+use Core\Infrastructure\Common\Api\HttpUrlTrait;
 use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
 use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
 
 final class HostGroupResponseNormalizer implements NormalizerInterface
 {
+    use HttpUrlTrait;
+    private const IMAGE_DIRECTORY = '/img/media/';
+
     public function __construct(
-        private readonly ObjectNormalizer $normalizer,
+        private readonly ObjectNormalizer $normalizer
```

Comment:
```suggestion
        private readonly string $mediaPath,
        private readonly ObjectNormalizer $normalizer
```

If you apply previous suggestion